frontend/src/app/components/wp-table/embedded/embedded-tables-macro.component.ts

```diff
@@ -28,10 +28,11 @@
 
 import {Component, ElementRef} from "@angular/core";
 import {WorkPackageTableConfigurationObject} from "core-components/wp-table/wp-table-configuration";
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
+
+export const wpEmbeddedTableEntrySelector = 'macro.embedded-table';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/be03a9ed2bd960efe9bf08e68a4088034cb73cd9/frontend/src/app/components/wp-table/embedded/embedded-tables-macro.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>